CONTRIBUTING.md

```diff
@@ -88,7 +88,25 @@ docker run --rm -it -v ${PWD}:/addons -w /addons tensorflow/tensorflow:nightly-c
 
 bazel test -c opt -k \
 --test_timeout 300,450,1200,3600 \
---test_output=errors \
+--test_output=all \
+//tensorflow_addons/...
+```
+
+#### Locally Testing GPU
+```bash
+docker run --runtime=nvidia --rm -it -v ${PWD}:/addons -w /addons tensorflow/tensorflow:custom-op-gpu make gpu-unit-test
```

Comment:
Perhaps it's recommend to use `bash tools/run_docker.sh -d gpu -c 'make gpu-unit-test'` for brevity？